engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -207,23 +207,20 @@ public Vector3f getViewingDirection() {
     }
 
     /**
-     * Get the gaze direction.
-     * @return the gaze direction, a quaternion.
+     * Get the viewing direction.
+     * @return the viewing direction, a quaternion.
      */
-    public Quat4f getGazeDirection() {
-        return gazeDirection;
+    public Quat4f getOrientation() {
+        return new Quat4f(viewingDirection,viewingAngle);
     }
 
     /**
-     Try to set the gaze direction. * Some cameras might be immovable, such as a VR camera, thus "request."
-     * @param gazeDirectionIn
+     Try to set the viewing direction. * Some cameras might be immovable, such as a VR camera, thus "request."
+     * @param direction
      */
-    public void requestSetGazeDirection(Quat4f gazeDirectionIn) {
-        gazeDirection = gazeDirectionIn;
-    }
-
-    public Vector3f getUp() {
-        return up;
+    public void requestSetOrientation(Quat4f direction) {
+        viewingDirection = direction.getAxis();
+        viewingAngle = direction.getAngle();
```

Comment:
So this is gone, and I have commented above on why and how I've refactored things.